openshift/4.0/templates/monitor/deploy-grafana.yaml

```diff
@@ -1,1298 +0,0 @@
-kind: Template
```

Comment:
If we're removing grafana and prometheus, have you deleted them from the tools namespace too?

Reply:
no, I'd do that after we agreed to remove them here.